packages/teleport-shared/src/node-handlers/node-to-jsx/index.ts

```diff
@@ -88,7 +88,11 @@ const generateElementNode = (
 ) => {
   const { dependencies, nodesLookup } = params
   const { elementType, children, key, attrs, dependency, events } = node.content
-  const elementTag = createJSXTag(elementType)
+  // Append app for only local dependencies
+  const elementTag =
```

Comment:
check lines 122-132 where we handle dependencies. based on the `dependencyHandling` mode we do some things that can overlap with this functionality. let's talk about it to have a single solution maybe for handling the names of the local dependencies